README_cmake.md

```diff
@@ -466,6 +464,8 @@ If the CMake version is lower than 3.18, the deprecated [`FindCUDA`][findcuda]
 module will be used instead. It reads the variable `CUDA_TOOLKIT_ROOT_DIR`
 instead of `CUDAToolkit_ROOT` above.
 
+TODO: update this section for OpenGLCompute, which needs some (but maybe not all) of this.
```

Comment:
Were you planning to get to this TODO before merging?

Reply:
Well, I *was*, but since OpenGLCompute is ~untestable on desktop Linux until we fix the LLVM-exports issues (see https://github.com/halide/Halide/issues/5627), I think it's probably more useful to leave these as TODO (but open an Issue to fix them once OpenGLCompute is testable again)...